10-norms-best-practices.md

```diff
@@ -0,0 +1,37 @@
+---
+guide: Employee Handbook
+title: Norms and Best Practices
+---
+
+### Tips for Success
+
+We recognize that it takes time to settle in and absorb everything that comes along with a new job. Here are a few tried-and-true suggestions from the team to help:
+
+* **Work Schedule:** We have team members in varying time zones and with different work schedules. We trust you to manage your time in a way that helps you fulfill your hours, get the job done, and do what you gotta do in other parts of your life. We believe you can do YOU and your work.
+* **Calendars:** To help everyone schedule meetings without a lot of back and forth, share your calendar and make sure it's up to date. Calendar tip: Go to calendar settings and enter your work hours. Your calendar will display this when people try to schedule meetings with you.
+* **Managing projects and priorities:** Some of us are juggling multiple projects at any given time. If you have any questions or concerns about priorities, please connect with your manager, project leads, or [People Ops](mailto:blossom@bloomworks.digital).
+* **Communication:** As a remote team, it's crucial that we share information in a timely and transparent way. We use Slack as a virtual office space to connect with others on the team. Feel free to say 'hi' or otherwise ping people there, just as you would if they were sitting at the desk next to you. We also have a number of different Slack channels to browse and join if you feel called. Here's a listing Rule of thumb: When in doubt, over-communicate.
+* **Remote work life:** Working remotely can be an adjustment if you're new to it. We recommend developing practices that support managing your time, space, and well-being, like: working a generally regular schedule, making sure to take little breaks from the screen, connecting with others on the team just because, and asking other remote folks about the tricks they use to stay productive and whole. We also encourage you to meet with fellow Bloomers if you live in the same area!
+* **Boundaries:** Your time is your time, but your time may not be my time. If you are messaging someone outside of their working hours, consider scheduling your message to be sent at a later time if it’s not urgent. Here is how you do it in [Slack](https://slack.com/help/articles/201457107-Send-and-read-messages#send-or-schedule-messages) and [Google](https://support.google.com/messages/answer/10456318?hl=en#zippy=%2Cset-a-reminder-for-a-message%2Cschedule-messages).
+
+While we can all get busy, we're always willing to try and help each other out. If you run into issues around work practices or project stuff, reach out to your manager sooner rather than later.
+
+**We'd love to hear other bits of related wisdom from the team, so please share below if you have additional tips we can integrate here!**
+
+<h2 id="how-we-use-slack">How we use Slack</h2>
```

Comment:
@SMakaiTakori There are two [anchor links](https://github.com/bloom-works/handbook/pull/35/files#diff-553a53be0be3fa3b0dfe28fccd2f423da514be900b9c8b6bcb3509dfb3421457R24) in this guide, so we'll need to include IDs for those two headings.

11ty supports Markdown plugins that let you format IDs like so: `## How we use Slack {#how-we-use-slack}`

But since those plugins aren't yet installed in our templates repo, I formatted these IDs as HTML to avoid build errors in the immediate future.